File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioDtoArgValueInput.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdComposite;
import com.neome.java.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioDtoArgValueInput extends StudioDtoArgValue
{
  @Nullable
  public MetaIdComposite compositeId;

  public MetaIdField fieldId;

  @Nullable
  public String[] valuePathArray;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoArgValueInput.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdComposite
import com.neome.api.meta.base.Types.MetaIdField

interface StudioDtoArgValueInput : StudioDtoArgValue {
    val compositeId: MetaIdComposite?
    val fieldId: MetaIdField
    val valuePathArray: List<String>?
}
